man/sniproxy.conf.5

```diff
@@ -113,6 +113,7 @@ improve performance.
 .nf
 listener 192.0.2.10:80 {
     protocol http
+    reuseport yes
```

Comment:
This is the first boolean configuration open we are introducing to the configuration syntax, do we want to use: true/false, yes/no, and/or on/off?